### DISCUSSION TITLE
Publishable form builder field templates

### DISCUSSION THREAD

**Discussion by DanjBethel** [filamentphp/filament#1838] [Ideas]
Not sure if I missed it in the documentation, but is there a way to customise the look and field of the form builder fields? Perhaps a publish command that publishes the templates for each field? Maybe that's asking for a bit much, but just wondered if it was possible.

**Comment by danharrin** [comment]
It's possible to override the views, but since these will likely make your app break when you update Filament in the future, we do not recommend it or document it.

We have `filament-*` CSS classes throughout the codebase. These can be used to safely tweak the styles in your form components. These classes are a recent addition, so we may be missing some that you need to use. Submit a PR if you need to add any more to help your customisation.
